### ISSUE TITLE
Infer ignore markers on iOS code

### ISSUE THREAD

**Issue by StefanoBiasu** [facebook/infer#812]
It's possible to have a special markers inside iOS to insert when a code snippet has not to be inspected by infer?

e.g.

> @implementation MyClass 
> 
> -(void)myMethod {
> 
> //inspected code
> NSString *prettyVariable = @"Pretty variable";
> 
> ###INFER-IGNORE
> 
> //uninspected code
> NSNumber *nullNumber = nil;
> NSArray *badArray = [NSArray arrayWithObjects:nullNumber, nil];
> 
> ###INFER-IGNORE
> 
> //inspected code
> NSString *anotherPrettyvariable = @"Another pretty variable";
> 
> }
> @end

**Comment by dulmarod** [comment]
No, we don't have support for that at the moment unfortunately, but if you want to silence NULL_DEREFERENCE, you can add Nonnull annotations.
